frontend/src/pages/projects/projectRoles/RoleLabelsSection.scss

```
@@ -0,0 +1,6 @@
+.odh-role-labels__header-spacer {
+  width: calc(
+    var(--pf-t--global--font--size--body--default) +
```

Stylelint (17.14.0)

[error] Unexpected newline after "+" (scss/operator-no-newline-after)

(scss/operator-no-newline-after)